release/README.md

```diff
@@ -22,28 +22,28 @@ Update `CHANGELOG.md` based on the release notes for Bigtable, Storage,
 Spanner, and the common libraries:
 
 ```bash
-# Summarize the output of this into CHANGELOG.md under the Bigtable header
+# Summarize the output of this into CHANGELOG.md under the "Bigtable" header
```

Comment:
No change needed, but I'll note that this section looks fairly automateable. I could imagine a small script like ./release/update_changelog.sh that does all this and leave the user with an edited CHANGELOG.md file that they can then manually further tweak and then send the PR.

Reply:
Ack.  I'll see what I can come up with.